### PR TITLE
Codesec secrets scan

### PR DESCRIPTION
**Pull request by anaghapamidi**
Removed the list of supported file types/extensions from secrets-scanning.adoc. 

@tsmithv11 - This change is a part of [this ticket](https://redlock.atlassian.net/browse/RLP-77343), can you please helps us validate if we scan all files matched to specific signatures?

### CODE REVIEW

**Comment by tsmithv11** [comment]
All of these changes look good. I would separate out the changes for secrets because we're waiting on approval. The rest can be added immediately.

**Comment by anaghapamidi** [comment]
> All of these changes look good. I would separate out the changes for secrets because we're waiting on approval. The rest can be added immediately.

@tsmithv11 - Should we park the edit for removal of the "supported extension list as well types/extensions in secrets-scanning.adoc." as well, until further notice? The other changes are updated.

File: code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc
Comment (Pubs-MV) [inline code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc:155]: @anaghapamidi did adding the xref in the UI path work?

Please confirm that you were able to run panconv and create a PDF without errors

Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc:155]: @Pubs-MV yes the xref in the UI path works and the PDF is being generated smoothly.

**Review by try-panwiac — APPROVED** [review]

**Review by tsmithv11 — APPROVED** [review]

**Comment by Pubs-MV** [comment]
@anaghapamidi please resolve the conflicts, so we can merge. Also add the tested label to confirm that the content builds.

**Comment by anaghapamidi** [comment]
Hi @Pubs-MV I have resolved the conflicts. Request your review and approval. 

cc: @tsmithv11

**Pull request merged**
